chroot-bin/croutoncycle

```diff
@@ -123,7 +123,7 @@ if [ "$tty" = 'tty1' ]; then
     for disp in $winlist; do
         if [ "${disp%"*"}" != "$disp" ]; then
             curdisp="$disp"
-            if [ -n "$xiwiactive" -a "${disp%"*"}" = "$aurawin" -a \
+            if [ -n "$xiwiactive" -a "$aurawin" != "${aurawin%${curdisp%"*"}*}" -a \
```

Comment:
Also, please keep to 80 characters.  Using disp instead of curdisp and moving the -a to the next line should fix that.